hack/build/operator/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM alpine:3.5
 
+RUN adduser -D etcd-operator
 RUN apk add --no-cache ca-certificates
 
 ADD _output/bin/etcd-operator /usr/local/bin
 ADD _output/bin/etcd-backup /usr/local/bin
 
+USER etcd-operator
```

Comment:
This is more than just etcd operator pod. We also need to think about:
- backup pod
- etcd pod